src/Search.jl

```diff
@@ -15,5 +15,5 @@ export
   ContextFreeEnumerator,
   ContextFreeBFSEnumerator,
   
-  enumerative_search
+  search
```

Comment:
should we also export `bfs_priority_function` and `bfs_expand_rules`?
I can think of a usecase in which someone wants bfs expand priority, but some other expansion principle.